services/frontend-service/src/ui/Pages/Locks/LocksPage.tsx

```diff
@@ -55,6 +57,36 @@ export const LocksPage: React.FC = () => {
             ),
         [envs]
     );
+    let teamLocks = useMemo(
+        () =>
+            sortLocks(
+                Object.values(envs)
+                    .map((env) =>
+                        Object.values(env.applications)
+                            .map((app) =>
+                                Object.values(app.teamLocks).map((lock) => ({
+                                    date: lock.createdAt,
+                                    environment: env.name,
+                                    team: app.team,
+                                    lockId: lock.lockId,
+                                    message: lock.message,
+                                    authorName: lock.createdBy?.name,
+                                    authorEmail: lock.createdBy?.email,
+                                }))
+                            )
+                            .flat()
+                    )
+                    .flat(),
+                'oldestToNewest'
```

Comment:
nitpick: I would prefer if this was not nested so deeply.
For example, we could have
```
const teamLocks = ...
return sortLocks(locks)
```